in_toto/gpg/formats.py

```diff
@@ -57,6 +57,21 @@
 import securesystemslib.schema as ssl_schema
 import securesystemslib.formats as ssl_formats
 
+def _create_pubkey_with_subkey_schema(pubkey_schema):
+  """Helper method to extend the passed public key schema with an optional
+  dictionary of sub public keys "subkeys" with the same schema."""
+  schema = pubkey_schema
+  subkey_schema_tpl =  ("subkeys", ssl_schema.Optional(
+        ssl_schema.DictOf(
+          key_schema=ssl_formats.KEYID_SCHEMA,
+          value_schema=pubkey_schema
+          )
+        )
+      )
+  # TODO: Find a way that does not require to access a proteced member
+  schema._required.append(subkey_schema_tpl) # pylint: disable=protected-access
```

Comment:
I don't quite understand this use of `_required()`.  The comment above stated that this is optional, but then this _required() call is here.